packages/flutter/lib/src/cupertino/text_field.dart

```diff
@@ -35,6 +35,11 @@ const Color _kSelectionHighlightColor = Color(0x667FAACF);
 const Color _kInactiveTextColor = Color(0xFFC2C2C2);
 const Color _kDisabledBackground = Color(0xFFFAFAFA);
 
+// An eyeballed value that moves the cursor slightly left of where it is
+// rendered by default so its positioning more accurately matches the native
```

Comment:
Describe where "where it is rendered by default" is. (presumably that position is a simpler/more useful position, thus answering the question of "why don't we just move the default rendering position to the current one -2 instead of doing something special for iOS")